lib/modules/showKarma.js

```diff
@@ -62,6 +70,32 @@ async function updateKarmaDiv() {
 	}
 }
 
+// If the user has gold and would like to, display the gilded-icon in the user span.
+async function updateUserSpan() {
+	// Get the user <span> located in the userbar
+	const userSpan = document.querySelector('#header-bottom-right .user');
+
+	if (userSpan) {
+		// Let's get the user's data.
+		const { data } = await getUserInfo();
+
+		// If the user has gold, display the icon and set the title text to the
+		// time remaining till it expires.
+		if (data.is_gold) {
+			const $gilded = $('<span>', { class: 'gilded-icon' });
+
+			if (data.gold_expiration) {
+				const today = new Date();
+				const expDate = new Date(data.gold_expiration * 1000); // s -> ms
+				$gilded.attr('title', `Until ${expDate.toDateString()} (${formatDateDiff(today, expDate)})`);
```

Comment:
Sure thing, I pulled that line from accountSwitcher (L231):

https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/lib/modules/accountSwitcher.js#L231

That may need to be modified as well to use `formatDate(expDate)` instead of `toDateString()`.